src/data/yarn/primary-options.js

```diff
@@ -0,0 +1,17 @@
+const options = [
+    { value: "add", label: "add" },
+    { value: "init", label: "init" },
+    { value: "install", label: "install" },
+    { value: "publish", label: "publish" },
+    { value: "remove", label: "remove" },
+  ];
+  
+  export const primaryOptions = options.sort((x, y) => {
+    if (x.value < y.value) {
+      return -1;
+    }
+    if (x.value > y.value) {
+      return 1;
+    }
+    return 0;
+  });
```

Comment:
Check this and coordinate with icy.

https://github.com/sliit-foss/cli-explorer/pull/48#discussion_r1702792098